be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/RollCallHandler.scala

```diff
@@ -72,8 +72,59 @@ sealed class RollCallHandler(dbRef: => AskableActorRef) extends MessageHandler {
   }
 
   def handleCloseRollCall(rpcMessage: JsonRpcRequest): GraphMessage = {
+    // We first create the three (independent) futures
```

Comment:
This comment is essentially repeating the code, which is rarely useful. (cf. code complete 2nd edition, section 32.4 "self-documenting code" / "keys to effective comments")

I don't think this code suffers heavily from lack of documentation, so don't stress it, but if we wanted to improve documentation a lot here, I would recommend: 

(A) having function API documentation, which explains what `handleCloseRollCall` is going to do (at API-abstraction levels)

(B) within the function, use comments when the code (even rewritten and clean) isn't good enough to convey your ideas and intents. Code Complete (32.5) provides a ton of good examples, but I'd roughly summarize it as:
- explaining the intent behind the code, "why" are we doing this. The code should be already clean & understandable, but explaining the programmer's intent in writing it often helps a lot when maintaining it. 
- providing "summaries" of the code base. "what" are we going to do in the following few lines (not how!)
- clarifying anything "surprising" to the reader, i.e. optimizations, odd bug workarounds (possibly with a link to related resources), justified disrespect of best practices, etc.